src/nlsolve/type.jl

```diff
@@ -6,6 +9,62 @@
     GLM
 end
 
+@enum NLStatus::Int8 begin
```

Comment:
Do these really belong here?

Reply:
Yes, it's specific to these solvers and internal to the package.